src/data/object_events/object_event_pic_tables.h

```diff
@@ -11,6 +11,437 @@ static const struct SpriteFrameImage sPicTable_LittleBoyCamper[] = {
     overworld_frame(gObjectEventPic_LittleBoyCamper, 2, 2, 8),
 };
 
+static const struct SpriteFrameImage sPicTable_Ruka[] = {
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 0),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 1),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 2),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 3),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 4),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 5),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 6),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 7),
+    overworld_frame(gObjectEventPic_Ruka, 2, 4, 8),
+};
+
+static const struct SpriteFrameImage sPicTable_Melissa[] = {
+    overworld_frame(gObjectEventPic_Melissa, 2, 4, 0),
```

Comment:
the shorthand is `ascending_overworld_frame(2,4, 8)` or something like that (it's in the file).